docs/release-notes/8.0.asciidoc

```diff
@@ -9,6 +9,7 @@
 [[known-issue-8.0.1]]
 ==== Known issues
 * An {endpoint-sec} integration bug prevents benign Windows files from being deleted under certain circumstances.
+* A new Lucene 9 validation change may cause errors whenever regular expressions are included in EQL queries. This bug affects users who upgrade from {stack} version 7.x to 8.x and are using event correlation rules. To resolve this issue, use triple quotes `""" """` for regular expressions in event correlation rule queries.
```

Comment:
It may be my own unfamiliarity with this, but "use triple quotes for regular expressions" wasn't clear to me -- I needed to dig into the originating SDH, look at some examples before I got it. Maybe "enclose in triple quotes" or similar?
```suggestion
* A new Lucene 9 validation change may cause errors whenever regular expressions are included in EQL queries. This bug affects users who upgrade from {stack} version 7.x to 8.x and are using event correlation rules. To resolve this issue, enclose regular expressions in triple quotes `""" """` in event correlation rule queries.
```

Reply:
This might work and it actually sounds clearer - @peluja1012 @e40pud what do you think?

Reply:
Sounds good to me!